selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/git/GitRevertCommit.java

```diff
@@ -52,61 +55,51 @@ public GitRevertCommit(SeleniumWebDriver seleniumWebDriver, Loader loader) {
   private WebElement cancelButton;
 
   public void waitRevertPanelOpened() {
-    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
-        .until(ExpectedConditions.visibilityOf(revertPanel));
+    seleniumWebDriverHelper.waitVisibility(revertPanel, ELEMENT_TIMEOUT_SEC);
   }
 
   public void waitRevertPanelClosed() {
-    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
-        .until(ExpectedConditions.invisibilityOfElementLocated(By.id(REVERT_COMMIT_PANEL)));
+    seleniumWebDriverHelper.waitInvisibility(By.id(REVERT_COMMIT_PANEL), ELEMENT_TIMEOUT_SEC);
   }
 
   public void clickRevertButton() {
-    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
-        .until(ExpectedConditions.visibilityOf(revertButton))
-        .click();
+    seleniumWebDriverHelper.waitAndClick(revertButton, ELEMENT_TIMEOUT_SEC);
   }
 
   public void clickCancelButton() {
-    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
-        .until(ExpectedConditions.visibilityOf(cancelButton))
-        .click();
+    seleniumWebDriverHelper.waitAndClick(cancelButton, ELEMENT_TIMEOUT_SEC);
     waitRevertPanelClosed();
   }
 
   public String getTopCommitRevision() {
     loader.waitOnClosed();
-    return new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
-        .until(ExpectedConditions.visibilityOf(revertPanel))
+    return seleniumWebDriverHelper
+        .waitVisibility(revertPanel, ELEMENT_TIMEOUT_SEC)
         .getText()
         .split("\n")[1]
         .replaceAll("\\.", "");
   }
 
   public String getTopCommitComment() {
     loader.waitOnClosed();
-    return new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
-        .until(ExpectedConditions.visibilityOf(revertPanel))
+    return seleniumWebDriverHelper
+        .waitVisibility(revertPanel, ELEMENT_TIMEOUT_SEC)
         .getText()
```

Comment:
Too complex procedure of getting revision number. IMHO usage of regex would make it more obvious.